build.gradle

```diff
@@ -90,6 +90,10 @@ subprojects {
         testImplementation "org.junit.jupiter:junit-jupiter:$junit5Version"
         testImplementation "org.junit.jupiter:junit-jupiter-params:$junit5Version"
         testImplementation "org.hamcrest:hamcrest-library:$hamcrestVersion"
+        testImplementation "org.mockito:mockito-core:3.+"
```

Comment:
Please use explicit versions, version ranges are a bad idea.
Write down the version in `gradle.properties`.